source/core/GenericFunctions.cpp

```diff
@@ -693,17 +697,19 @@ InstructionCore* EmitMaxMinEltsInstruction(ClumpParseState* pInstructionBuilder)
 static bool GetMinimumArrayDimensions(std::vector<TypedArrayCoreRef> arrays,
     ArrayDimensionVector* newDimensionLengths, IntIndex* newRank)
 {
-    int arrayCount = arrays.size();
-    for (int i = 0; i < arrayCount; i++) {
+    size_t arrayCount = arrays.size();
```

Comment:
I think this refactor is unrelated to the main issue. Consider separating this code change in an independent pull request.

Reply:
It is only a warning fix for unused code which I fixed while touching this file.
Seemed easier just to slip it in, as it's a no-op and shouldn't really need review.